### PR TITLE
Fixed margin around user notification for login modal

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

Commit with this fix didn't make it into my last PR before it was merged.

## Screenshot

**Before**
screen shot 2016-08-15 at 1 48 52 pm

**After**
selection_654

### CODE REVIEW

File: kolibri/core/assets/src/vue/session-nav-widget/login-modal.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <div>
     <modal>
-      <div class="title" aria-labelledby="loginModal" slot="header">
+      <div class="title" aria-label="Log in to Kolibri" slot="header">
```
Comment (radinamatic) [inline kolibri/core/assets/src/vue/session-nav-widget/login-modal.vue]: @rtibbles This is the only change that will require string-wrapping, but at this point I'm not sure if I should wait for your PR to be merged, try wrapping only this myself or something else...?

**Comment by indirectlylit** [comment]
speaking on behalf of @rtibbles :) we can wrap the string as another task

**Comment by radinamatic** [comment]
I must be (really) slow tonight, but does that mean I should try to wrap it now (as another task) or at some later point? :blush: 

@indirectlylit

**Comment by indirectlylit** [comment]
at some later point :) not critical for today

One comment/question on the login screen, which is partly A11Y and partly design:

Is it necessary to have the big **Log In Error!** text? It feels loud and unnecessary to me.

**Comment by radinamatic** [comment]
It's big because it's wrapped inside `H1` tag, but I can easily style it to be smaller!

**Comment by radinamatic** [comment]
1.2em instead of 1.5em:

selection_655

Should I go even smaller? @indirectlylit

**Comment by indirectlylit** [comment]
Can we go all the way to deleted? :) (It seems unnecessary to me)

**Comment by indirectlylit** [comment]
It's also odd to me that the dialog has no `h1` tag until/unless they have a log-in error. Seems the dialog should have an `h1` with "Log In to Kolibri" or similar, and the error is a secondary element of some kind?

**Comment by radinamatic** [comment]
Putting the user notification into main heading is the [first recommended WAI technique](https://www.w3.org/WAI/tutorials/forms/notifications/#using-the-main-heading) for error information.

It is a good point that there's no H1 unless there is an error, but the `div.title` has an `aria-label` so I think we are covered there. We can make that **Log In Error!** look as the normal text, or apply the `visuallyhidden` class if you prefer, but IMHO the H1 should remain.

**Comment by indirectlylit** [comment]
Interesting. I'm slowly learning that accessibility and semantically-correct HTML are sometimes in opposition. (we touched on this with headers in the cards also). Okay, let's leave it there.

There's a minor thing still that I can't quite put my finger on about the wording though - a combination of the phrase "Log In Error" and the double exclamation marks...

Maybe one of either 'Log-in Error' or just 'Error' would suffice?

**Comment by radinamatic** [comment]
I'm all for better wording suggestions: "Log-in Error" as `H1` seems fine to me... :smiley:

selection_657

Ref. the **opposition** you mention, I'm more inclined to think in terms of knowing how to best harness the advantages of semantic elements: sometimes it can be completely in line with the "correct", and sometimes it's about knowing how the actual people use them (like preferring the headings for easier browsing of search results). 

Ref. the headings themselves, we still have some work to do:

selection_656

There should be some kind of `v-if` on that **Whoops!**... :stuck_out_tongue:

**Comment by indirectlylit** [comment]
"Whoops!" - https://github.com/learningequality/kolibri/commit/9fb6d7e96cfbbabc89b59a56f4e6c07ef69da718

**Pull request merged**